app/partials/request.jade

```diff
@@ -13,12 +13,12 @@
       .flex-center.flex-justify.auto(ng-show="focus")
         span.type-lbl {{ fields.to.label }}
       .flex-1(ng-hide="focus")
-        ui-select(ng-model="fields.to" ng-disabled="paymentRequest.amount > 0 || paymentRequest.label")
-          ui-select-match(placeholder="{{'SEARCH' | translate}}...")
+        ui-select(ng-model="fields.to" ng-disabled="paymentRequest.amount > 0 || paymentRequest.label" tagging)
+          ui-select-match(placeholder="{{::'SEARCH' | translate}}...")
             label-origin(origin="$select.selected")
-          ui-select-choices(repeat="destination in destinations | filter: getFilter($select.search)" group-by="'type'")
-            span
-              label-origin(origin="destination" highlight="$select.search")
+          ui-select-choices(repeat="destination in destinations | filter: {label:$select.search} | limitTo:destinationLimit" group-by="'type'")
+            span(in-view="$last && destination.type==='Imported Addresses' && increaseLimit()")
```

Comment:
Good catch. It's supposed to increase the number of list items when it reaches the last imported address. I didn't think about the case where the user has >50 accounts and 0 addresses. Probably an edge case, but will commit a fix for it.